testing/singularity/APKBUILD

```diff
@@ -17,7 +17,7 @@ makedepends="
 	libuuid
 	util-linux-dev
 	libseccomp-dev
-	"
+	cryptsetup"
```

Comment:
please keep the double-quotes in a separate line

Reply:
DOne. Thank you!